projects/igniteui-angular/src/lib/grids/grid-base.directive.ts

```diff
@@ -2150,7 +2184,16 @@ export class IgxGridBaseDirective extends DisplayDensityBase implements
     }
 
     /**
-     *@hidden
+     * Emitted before sorting is performed through the UI. Returns the sorting expressions.
```

Comment:
The event is emitted no matter if it is UI or API change.

Reply:
Fixed.